### DISCUSSION TITLE
vscode setup

### DISCUSSION THREAD

**Discussion by Tejakrishna2003** [Cyfrin/foundry-full-course-cu#2220] [General]
In vscode setup you have mentioned using GitHub copilot is there any alternative now every AI platform is charging some amount so suggest any free platforms .

**Comment by Alessandro-Cavaliere** [comment]
Hello, you can get GitHup Copilot for free if you are a student and provide your student data, otherwise there are several alternatives if you search online, I know llama for vscode, but I've never tried it and I can't give you an objective opinion.

If you search online I think you will find what you need.

**Reply by EngrPips** [reply]
Yeah, I heard of the Free Copilot if you are a student.

**Reply by Alessandro-Cavaliere** [reply]
> Yeah, I heard of the Free Copilot if you are a student.

yep, I have this one ;)

**Reply by EngrPips** [reply]
I graduated few years back lol, and I am still a student of Patrick though I am not sure if that will suffice to claim the privilege lol

**Comment by urosognjenovic** [comment]
I've been using Codeium for quite some time now and I have no complaints about it so far.

**Reply by EngrPips** [reply]
I think I should try out this Codeium.
